plugins/polio/js/src/domains/VaccineModule/SupplyChain/Table/useVaccineSupplyChainTableColumns.tsx

```diff
@@ -68,6 +68,17 @@ export const useVaccineSupplyChainTableColumns = (): Column[] => {
                     return <span>{rounds.map(r => r.number).join(', ')}</span>;
                 },
             },
+            {
+                Header: formatMessage(MESSAGES.dosesRequested),
+                accessor: 'quantities_ordered_in_doses',
+                Cell: settings => (
+                    <NumberCell
+                        value={
+                            settings.row.original.quantities_ordered_in_doses
+                        }
+                    />
+                ),
```

Comment:
you can properly type this like this 
`                Cell: ({
                    row: { original },
                }: ColumnCell<OrgUnitChangeRequest>): ReactElement => {
`

Reply:
something similar i mean